Replicator/Replicator.cc

```diff
@@ -85,6 +85,7 @@ namespace litecore { namespace repl {
 
     void Replicator::_stop() {
         logInfo("Told to stop!");
+        _dbActor->flush();
```

Comment:
You can't call an actor method like this directly -- you're on the wrong thread/queue. `_markRevsSyncedNow` uses internal state, so it has to run on the DBWorker's actor queue. But that means the call will be asynchronous, which won't achieve the goal of running it before the replicator finishes. A different solution is necessary.